usecases/auth/authorization/mocks/AuthorizerMock.go

```diff
@@ -46,6 +47,31 @@ func (_m *Authorizer) Authorize(principal *models.Principal, verb string, resour
 	return r0
 }
 
+// AuthorizeSilent provides a mock function with given fields: principal, verb, resources
+func (_m *Authorizer) AuthorizeSilent(principal *models.Principal, verb string, resources ...string) error {
```

Comment:
should we just call the function above? seems identical and there is no logging anyways

Reply:
this is generated code